src/snapred/backend/recipe/algorithm/CalibrationReductionAlgorithm.py

```diff
@@ -33,16 +37,16 @@ def PyExec(self):
         raw_data = self.mantidSnapper.LoadEventNexus(
             "Loading Event Nexus for {} ...".format(rawDataPath),
             Filename=rawDataPath,
-            FilterByTofMin=2000,
-            FilterByTofMax=14500,
+            FilterByTofMin=self.TOF_MIN,
+            FilterByTofMax=self.TOF_MAX,
             OutputWorkspace="raw_data",
         )
 
         self.mantidSnapper.Rebin(
             "Rebinning to 2000 bins logarithmic...",
             InputWorkspace=raw_data,
             OutputWorkspace=raw_data,
-            Params="2000,-0.001,14500",
+            Params=self.REBIN_PARAMS,
```

Comment:
I'm not sure how config renders this, but I just want to confirm that this is being passed as a tuple (or list) and not a string

Reply:
Yes this is being passed as a list. I tried forcing it as a tuple but the pre-commit tools did not like the python tag `!!python/tuple`. I couldn't find any other clean and simple ways of forcing a tuple.